index.d.ts

```diff
@@ -1832,7 +1836,7 @@ declare namespace Eris {
     getDiscovery(): Promise<DiscoveryMetadata>;
     /** @deprecated */
     getEmbed(): Promise<Widget>;
-    getIntegrations(): Promise<GuildIntegration>;
+    getIntegrations(options: GetGuildIntegrationsOptions): Promise<GuildIntegration>;
```

Comment:
Param should be optional
```suggestion
    getIntegrations(options?: GetGuildIntegrationsOptions): Promise<GuildIntegration>;
```